.github/workflows/docker.yml

```diff
@@ -0,0 +1,115 @@
+name: docker
+
+on:
+  schedule:
+    - cron: '0 10 * * *' # everyday at 10am
+  push:
+    branches: [ master ]
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  start-runner:
```

Comment:
Why is this running on a self-hosted runner? Because GH runner have too small disks?

Reply:
@tgaddair saw your comment above.

Reply:
Yeah, they run out disk pretty quickly, as the image is over 10GB.